init.lua

```diff
@@ -3,13 +3,14 @@ local server = require("server")
 local hud = require("hud")
 local event = require("event")
 local powerControl = require("modules.tools.powerControl")
+local moduleSelection = require("configuration.modules")
 --
 
 local serverInfo
 
 while true do
     serverInfo = server.update()
-    hud.update(serverInfo)
-    powerControl(serverInfo.power, "b8583fd9")
+    if moduleSelection.hud then hud.update(serverInfo) end
```

Comment:
If the user does not want a module to run, it shouldn't even be loaded.
`require(hud)` on a tablet breaks the system.
Check feat/tablet-setup